client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { map } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+
+// Mapping eligibility holds to messages that will be shown to the user
+// TODO: update supportUrls and maybe create similar mapping for warnings
+function getHoldMessages( translate ) {
+	return {
+		PLACEHOLDER: {
+			title: '',
+			description: '',
+			supportUrl: '',
+		},
+		TRANSFER_ALREADY_EXISTS: {
+			title: translate( 'Installation in progress' ),
+			description: translate( 'Another installation is already in progress.' ),
+			supportUrl: 'https://wordpress.com/help'
+		},
+		NO_BUSINESS_PLAN: {
+			title: translate( 'Business plan required' ),
+			description: translate( 'This feature is only allowed on sites with a business plan.' ),
+			supportUrl: 'https://support.wordpress.com/'
+		},
+		NO_JETPACK_SITES: {
+			title: translate( 'Jetpack site' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **16** times:
`translate( 'Manage Jetpack Sites' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).